contributions/catalog/controllers/config.py

```diff
@@ -19,25 +19,4 @@ class Config(object):
     client_id = os.getenv("CLIENT_ID", "NO ID")
     client_secret = os.getenv("CLIENT_SECRET", "NO SECRET")
     authorization_base_url = 'https://github.com/login/oauth/authorize'
-    token_url = 'https://github.com/login/oauth/access_token'
-
-    # # ADMINS defines a list usernames that serve as administrators
-    # ADMINS = []
-    #
-    # # OIDC CONFIG FOR LOGIN
-    # ISSUER_URL = os.getenv("ISSUER_URL", "https://shibboleth-test.techservices.illinois.edu")  # test instance
-    # SCOPES = os.getenv("SCOPES",
-    #                    ["openid", "profile", "email", "offline_access"])  # Other OIDC scopes can be added as needed.
-    # REDIRECT_URIS = os.getenv("REDIRECT_URIS", "")
-    # CLIENT_ID = os.getenv("CLIENT_ID", "")
-    # CLIENT_SECRET = os.getenv("CLIENT_SECRET", "")
-    #
-    # # LOGIN_MODE SELECTED
-    # LOGIN_MODE = os.getenv("LOGIN_MODE", "shibboleth")  # TODO: Add an option "local" for local login.
-    #
-    # # ROLE OF USERS
-    # ROLE = {
-    # }
-    #
-    # # SESSION EXPIRATION TIME
-    # PERMANENT_SESSION_LIFETIME = timedelta(hours=os.getenv("SESSION_LIFETIME_IN_HOURS", 1))
+    token_url = 'https://github.com/login/oauth/access_token'
```

Comment:
Can we not take a similar approach to using dotenv as in App Config and other existing building blocks? I.e., calling load_dotenv() method in config.py in the begenning, there by giving preference to values in .env. Currently, there is an error when some variables like `authorization_base_url` are not in .env file. This way, it'll get those values directly from config.py. 

https://github.com/rokwire/rokwire-building-blocks-api/blob/develop/appconfigservice/api/controllers/config.py